cypress/support/pages/admin/pageAdministration.js

```diff
@@ -643,42 +663,64 @@ class Management {
 
 	clickOnAddUserInFAB(role) {
 		let addUserButtonInFAB =
-			role === "student" ? Management.#addStudentButton : Management.#addTeacherButton;
+			role === "student"
+				? Management.#addStudentButton
+				: Management.#addTeacherButton;
 		cy.get(addUserButtonInFAB).click({ force: true });
 	}
 
 	fillUserCreationForm(forename, surname, baseEmail) {
 		const randomNumber = new Date().getTime() + Math.floor(Math.random() * 1000);
 		const uniqueEmail = randomNumber + baseEmail;
 
-		// Store the generated unique email as an alias for later use
+		// store the generated unique email as an alias for later use
 		cy.wrap(uniqueEmail).as("uniqueEmail");
-
-		// Log the generated unique email
 		cy.log("Generated Unique Email:", uniqueEmail);
 
-		// Fill in the form with the generated email and other details
+		// fill in the form with the generated email and other details
 		cy.get(Management.#firstNameCreationForm).type(forename);
 		cy.get(Management.#lastNameCreationForm).type(surname);
 		cy.get(Management.#emailCreationForm).type(uniqueEmail);
 
-		// Setting the birth date to 17 years ago in the form
+		// setting the birth date to 17 years ago in the form for student user
 		cy.get("body").then((body) => {
 			if (body.find(Management.#birthDateFieldCreateStudent).length) {
-				// For Student
 				const birthDate = new Date();
 				birthDate.setFullYear(birthDate.getFullYear() - 17);
-				cy.get(Management.#birthDateFieldCreateStudent).type(
-					birthDate.toISOString().split("T")[0],
-					100
-				);
+
+				// YYYY-MM-DD for typing into the input
+				// note: toISOString() returns date in UTC, so we split and take only the date part
+				// this ensures that the date is correctly interpreted regardless of local timezone
+				const isoDate = birthDate.toISOString().split("T")[0];
+				const formattedBirthDate = `${String(birthDate.getDate()).padStart(2, "0")}.${String(birthDate.getMonth() + 1).padStart(2, "0")}.${birthDate.getFullYear()}`; // DD.MM.YYYY for table check
+
+				// type the ISO string into the input (works for <input type="date">)
+				cy.get(Management.#birthDateFieldCreateStudent).type(isoDate, 100);
+
+				// store alias in DD.MM.YYYY format for the assertion in user table
+				cy.wrap(formattedBirthDate).as("assignedBirthDate");
 			} else {
-				//For Teacher
+				// for Teacher
 				cy.log("Birthdate is not required while creating a new teacher");
 			}
 		});
 	}
 
+	seeTheAssignedBirthDateInUserTable() {
+		// check if the alias exists before trying to get it
+		if (Cypress._.get(cy.state("aliases"), "assignedBirthDate")) {
+			cy.get("@assignedBirthDate").then((assignedBirthDate) => {
+				cy.log("verifying assigned birthdate for student:", assignedBirthDate);
+				cy.get(Management.#tableContents)
+					.should("exist")
+					.and("contain", assignedBirthDate);
+			});
+		} else {
+			// for teacher
+			cy.log("user is not a student");
```

Comment:
Why reference the role here? It is at least not really correct from a general point of view because there can be students without a birthday.